18F/identity-idp
# LG-13795: Deemphasize backup codes

## 🎫 Ticket

[LG-13795](https://cm-jira.usa.gov/browse/LG-13795)

## 🛠 Summary of changes

Updates backup codes:

- Reorders MFA setup options to list backup codes as the last option
- Updates backup codes setup description to discourage use
- Reduces number of backup codes generated from 10 to 5

~**Draft** pending confirmation of reduction of number of backup codes ([see Slack discussion](https://gsa-tts.slack.com/archives/C01710KMYUB/p1721415619436179))~ Per Slack discussion, keeping count to 10.

## 📜 Testing Plan

1. Go to http://localhost:3000
2. Click "Create an account"
3. Continue account creation until MFA selection
4. Observe that "Backup codes" is listed last, and has updated description text
5. Select "Backup codes"
6. Click "Continue"
7. Observe that you are given 5 backup codes instead of 10

## 👀 Screenshots

Screen|Before|After
---|---|---
MFA Setup|![before](https://github.com/user-attachments/assets/52fbd605-e104-4603-bd39-930b5c490b0e)|![after](https://github.com/user-attachments/assets/56ee3c6e-8137-4d5e-81ad-ed8ed759e8ed)
Backup codes|![image](https://github.com/user-attachments/assets/9015e9e7-d5f3-4d62-a920-002afe00b31e)|![image](https://github.com/user-attachments/assets/e7f60163-0ad0-4cc2-8f49-cb940e5c812b)
